redbot/cogs/trivia/data/lists/geography.yaml

```diff
@@ -0,0 +1,213 @@
+AUTHOR: Kreusada#0518
+What is Earth's largest continent?:
+- Asia
+What country accounts for more than half of the western coastline of South America?:
+- Chile
+What river runs through Baghdad?:
+- Tigris
+- Tigris River
+What country has the most natural lakes?:
+- Canada
+What is the only sea without any coasts?:
+- Sargasso
+- Sargasso Sea
+What percentage of the River Nile is located in Egypt?:
+- 22%
+What is the driest place on Earth?:
+- McMurdo
+- McMurdo Dry Valleys
+- McMurdo Valleys
+In what country can you visit Machu Picchu?:
+- Peru
+Which African nation has the most pyramids?:
+- Sudan
+What African country served as the setting for Tatooine in Star Wars?:
+- Tunisia
+What is the oldest city in the world?:
+- Damascus
+Which U.S. state has the most active volcanoes?:
+- Alaska
+What is the flattest continent?:
+- Australia
+What is the largest country on the Arabian Peninsula?:
+- Saudi Arabia
+What country has the most coastline?:
+- Canada
+"Which of the following volcanoes is NOT located in Antarctica?\nA - Mount Melbourne\nB - Mount Berlin\nC - Mount Hampton\nD - Mount Napier":
+- Mount Napier
+- D
+What country is home to Kangaroo Island?:
+- Australia
+What is the largest country in South America?:
+- Brazil
+What is the tallest mountain in the world?:
+- Mount Everest
+- Everest
+What is the capital of Australia?:
+- Canberra
+Riyadh is the capital of what Middle-Eastern country?:
+- Saudi Arabia
+What is the smallest independent country on Earth?:
+- Vatican City
+What is the deepest point in Earth's oceans?:
+- Mariana Trench
+What mountain is nicknamed the "Savage Mountain"?:
+- K2
+Along with Spain and France, what is the other country to have both Atlantic and Mediterranean coastlines?:
+- Morocco
+What country occupies the southern end of the Arabian Peninsula?:
+- Yemen
+What South American country shares the bulk of the Southern Cone with its neighbor Chile to the west?:
+- Argentina
+What is the tallest mountain in North America?:
+- Denali
+- Mount McKinley
+- McKinley
+Montevideo is the capital of what South American country?:
+- Uruguay
+What U.S. state shares borders with Louisiana, Arkansas, Oklahoma, and New Mexico?:
+- Texas
+What is the only continent with land in all four hemispheres?:
+- Africa
+What mountain is closest to the Moon?:
+- Mount Chimborazo
+- Chimborazo
+What continent contains the most fresh water?:
+- Antarctica
+What ocean is home to 75% of the Earth's volcanoes?:
+- Pacific 
+- Pacific Ocean
+What is the largest city in the world based on surface area?:
+- Hulunbuir
+What is the only major city located on two continents?:
+- Istanbul
+What is the coldest sea on Earth?:
+- White Sea
+- White
+What is America's largest city by surface area?:
+- Yakutat
```

Comment:
This is listed as a [borough](https://en.wikipedia.org/wiki/Yakutat,_Alaska), not a city. The largest city is supposedly [Sitka](https://en.wikipedia.org/wiki/List_of_United_States_cities_by_area).